README.md

```diff
@@ -27,6 +27,99 @@ https://github.com/FormidableLabs/builder-init/issues/6
 
 ## Templates
 
+### Archetype Data
```

Comment:
I was confused about what this project was for from the main description. It would be helpful to better explain that this is for using archetypes to generate projects and not for generating archetypes, maybe with a use case example?